upup/models/cloudup/resources/addons/aws-ebs-csi-driver.addons.k8s.io/k8s-1.17.yaml.template

```diff
@@ -395,6 +395,10 @@ spec:
         app.kubernetes.io/name: aws-ebs-csi-driver
         app.kubernetes.io/instance: aws-ebs-csi-driver
         app.kubernetes.io/version: {{ .Version }}
+      annotations:
```

Comment:
this should be `range` https://github.com/kubernetes/kops/pull/13536/files#diff-387221074177e2de294fd64375019fcfcae1519aebd3b97d2c9dc7ae9426a53fR131

Reply:
ok